codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexCodecTest.java

```diff
@@ -170,6 +170,16 @@ public void inboundDataFrameShouldEmitWindowUpdateFrame() {
         verifyFramesMultiplexedToCorrectChannel(inboundStream, inboundHandler, 2);
     }
 
+    @Test
+    public void unhandledHttp2FramesShouldBePropagated() {
+        ByteBuf content = UnpooledByteBufAllocator.DEFAULT.buffer(8).writeLong(0);
+        Http2PingFrame decodedFrame = new DefaultHttp2PingFrame(content);
+
+        codec.onHttp2Frame(decodedFrame);
+        Http2PingFrame receivedPing = parentChannel.readInbound();
+        assertSame(receivedPing, decodedFrame);
```

Comment:
you need to call `receivedPing.release()` as well to ensure we not leak. This way the CI will also pass.

Reply:
I think you might have been looking at an outdated PR: it's released on line 181 and tests are green.

Reply:
yep seems so :D all good...